api/namex/constants/__init__.py

```diff
@@ -669,3 +669,7 @@ class EventUserId(AbstractEnum):
 class RequestPriority(AbstractEnum):
     Y = 'Y'
     N = 'N'
+
+class ExpiryDays(AbstractEnum):
```

Comment:
if you change to ```(str, AbstractEnum)```
then you can compare to a string value, makes it a little easier to use.

Reply:
updated this, and i also updated version file (i forget that needs to be done also)